chromiumcontent/chromiumcontent.gypi

```diff
@@ -122,7 +122,7 @@
           },
         },
       }],
-      ['_target_name in ["gtk2ui", "devtools_http_handler", "devtools_discovery", "http_server"]', {
+      ['_target_name in ["gtk2ui", "devtools_http_handler", "devtools_discovery", "http_server", "desktop_capture", "webrtc_common", "system_wrapper", "rtc_base", "rtc_base_approved", "libyuv"]', {
```

Comment:
We do need it here to guarantee these static libraries are built as stand-alone libraries, otherwise most of the static libraries in Chromium are created as a `thin archive`(at least on Linux) to reduce the size, which will cause the undefined symbol error during `Electron's` linking.